test/make_dist.py

```diff
@@ -131,9 +131,9 @@ def download_dist(wait=False):
         tar_path = os.path.realpath(names[0])
 
     # Extract node_modules and dist locally for speeding up the build and allowing integration tests to run
```

Comment:
Do you want to add package-lock.json in the comment here as well? 
And maybe rename the unpack_dirs to unpack_items? or something else. Jist nitpicking here I don't care so much about it.

Reply:
Right. I did that in the split out PR #15727.